### PR TITLE
[snap] add network-bind and inputstream-adaptive

### PR DESCRIPTION
**Pull request by Saviq**

### CODE REVIEW

File: snap/snapcraft.yaml [hunk 1]
```diff
@@ -99,6 +100,7 @@ parts:
       - kodi-wayland
       - kodi-visualization-spectrum
       - kodi-repository-kodi
+      - kodi-inputstream-adaptive
```
Comment (AlanGriffiths) [inline snap/snapcraft.yaml]: Which repos contain this? It doesn't seem to be in the 20.04 archive - which we use for arm64/armhf.

Comment (Saviq) [reply snap/snapcraft.yaml]: Right… I've `ppa:team-xbmc` enabled… but it's amd64 only. I suppose we could have this `try: ['kodi-inputstream-adaptive']` and add the PPA?

Comment (AlanGriffiths) [reply snap/snapcraft.yaml]: We already add the PPA (for Intel), I suggest:
```
    - on amd64: kodi-inputstream-adaptive
    - on i386: kodi-inputstream-adaptive
```
With, maybe, some comments.

Comment (Saviq) [reply snap/snapcraft.yaml]: OK with this?

```suggestion
      - try: [kodi-inputstream-adaptive] # only available for x86
```

File: snap/snapcraft.yaml [hunk 2]
```diff
@@ -99,6 +100,7 @@ parts:
       - kodi-wayland
       - kodi-visualization-spectrum
       - kodi-repository-kodi
+      - try: [kodi-inputstream-adaptive] # only available for x86
```
Comment (AlanGriffiths) [inline snap/snapcraft.yaml]: I've a mild preference for saying exactly what should work over trying and ignoring failures, but :man_shrugging:

**Review by AlanGriffiths — APPROVED** [review]
bors d=Saviq

**Review by AlanGriffiths — APPROVED** [review]
LGTM

bors r+

**Pull request merged**